ingestion/functions/parsing/argentina/argentina.py

```diff
@@ -0,0 +1,233 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+
+def convert_date(date_str: str):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYYmmdd format, but the data server API will
+    assume that ambiguous cases (e.g. "05/06/2020") are in mm/dd/YYYY format.
+
+    Adding line to ensure date has type str
+    """
+    date = datetime.strptime(date_str, "%Y-%m-%d")
+
+    return date.strftime("%m/%d/%Y")
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "M":
+        return "Male"
+    if raw_gender == "F":
+        return "Female"
+    return None
+
+
+def convert_location(entry):
+    '''
+    The only information we have is the province where case was diagnosed/hospitalised
+    '''
+
+    if entry['carga_provincia_nombre']:
+        return {"query": f"{entry['carga_provincia_nombre']}, Argentina"}
+    else:
+        return {"query": "Argentina"}
+
+
+def convert_age(entry):
+    '''
+    Want to return a float specifying age in years. If age field is empty, return None
+    '''
+    if entry['edad']:
+        if entry['edad_años_meses'] == 'Años':
+            return float(entry['edad'])
+        elif entry['edad_años_meses'] == 'Meses':
+            return float(float(entry['edad']) / 12)
```

Comment:
are the ages in the source given as floats? I'd expect int instead.

Reply:
yes they're ints, but don't we want to convert them to float for input?

Reply:
`float(int(entry['edad']) / 12` is what I meant, sorry if I wasn't clear